classical/airport-adl/domain.pddl

```diff
@@ -9,7 +9,7 @@
 
 
 (define (domain airport)
-(:requirements :adl)
```

Comment:
This type of change seems like a set-back. Can you detail the protocol to remove `:adl` in lieu of the several smaller ones?

Reply:
Was using:
```python
ADL_REQS = {":strips",
    ":typing",
    ":disjunctive-preconditions",
    ":equality",
    ":quantified-preconditions",
    ":conditional-effects"
}
```
:strips shouldn't be there I guess, but also `quanitfied-precondition` maybe is an alias?

I got the conditions from https://planning.wiki/ref/pddl/requirements#adl

Reply:
`:quantified-preconditions` doesn't appear in the classical-domains at all btw